module/migration.mjs

```diff
@@ -477,7 +477,10 @@ export function migrateItemData(item, itemData, migrationData, flags={}) {
 
   // Migrate embedded effects
   if ( itemData.effects ) {
-    const effects = migrateEffects(itemData, migrationData);
+    if ( itemData.flags?.dnd5e?.riders?.effect ) {
+      itemUpdateData["flags.dnd5e.riders.effect"] = itemData.flags.dnd5e.riders.effect;
+    }
+    const effects = migrateEffects(itemData, migrationData, updateData);
```

Comment:
Need to make sure items with this rider flag don't get migrated every time, since we set `diff: false` for these updates.

```suggestion
    const riders = foundry.utils.getProperty(itemData, "flags.dnd5e.riders.effect");
    if ( riders?.length ) updateData["flags.dnd5e.riders.effect"] = riders;
    const effects = migrateEffects(itemData, migrationData, updateData);
    if ( riders?.length === updateData["flags.dnd5e.riders.effect"]?.length ) delete updateData["flags.dnd5e.riders.effect"];
```